src/test/java/com/powsybl/openloadflow/equations/LUDecompositionTest.java

```diff
@@ -0,0 +1,132 @@
+package com.powsybl.openloadflow.equations;
```

Comment:
Could you add the licence header and authorship ?

Reply:
Done